sources/packages/web/src/main.ts

```diff
@@ -33,29 +33,32 @@ import FooterButtons from "@/components/generic/FooterButtons.vue";
 import TitleValue from "@/components/generic/TitleValue.vue";
 import StatusInfoLabel from "@/components/generic/StatusInfoLabel.vue";
 
-AppConfigService.shared.init().then(() => {
-  createApp(App)
-    .use(vuetify)
-    .use(store)
-    .use(router)
-    .use(PrimeVue)
-    .component("DataTable", DataTable)
-    .component("Column", Column)
-    .component("BodyHeader", BodyHeader)
-    .component("ContentGroup", ContentGroup)
-    .component("ContentGroupInfo", ContentGroupInfo)
-    .component("TooltipIcon", TooltipIcon)
-    .component("StatusInfoLabel", StatusInfoLabel)
-    .component("FullPageContainer", FullPageContainer)
-    .component("TabContainer", TabContainer)
-    .component("BodyHeaderContainer", BodyHeaderContainer)
-    .component("StudentPageContainer", StudentPageContainer)
-    .component("HeaderNavigator", HeaderNavigator)
-    .component("Banner", Banner)
-    .component("formio", formio)
-    .component("ToggleContent", ToggleContent)
-    .component("FormioContainer", FormioContainer)
-    .component("FooterButtons", FooterButtons)
-    .component("TitleValue", TitleValue)
-    .mount("#app");
-});
+AppConfigService.shared
+  .init()
+  .then(() => {
+    createApp(App)
+      .use(vuetify)
+      .use(store)
+      .use(router)
+      .use(PrimeVue)
+      .component("DataTable", DataTable)
+      .component("Column", Column)
+      .component("BodyHeader", BodyHeader)
+      .component("ContentGroup", ContentGroup)
+      .component("ContentGroupInfo", ContentGroupInfo)
+      .component("TooltipIcon", TooltipIcon)
+      .component("StatusInfoLabel", StatusInfoLabel)
+      .component("FullPageContainer", FullPageContainer)
+      .component("TabContainer", TabContainer)
+      .component("BodyHeaderContainer", BodyHeaderContainer)
+      .component("StudentPageContainer", StudentPageContainer)
+      .component("HeaderNavigator", HeaderNavigator)
+      .component("Banner", Banner)
+      .component("formio", formio)
+      .component("ToggleContent", ToggleContent)
+      .component("FormioContainer", FormioContainer)
+      .component("FooterButtons", FooterButtons)
+      .component("TitleValue", TitleValue)
+      .mount("#app");
+  })
+  .catch((e) => e);
```

Comment:
What is the purpose of the `catch` here?

Reply:
Callback when the promise is rejected

Reply:
I see that, but it is not doing anything, right?

Reply:
Yes actually. I can log it or I can remove it but if I remove it, then Sonar complains

Reply:
Removed it as per the discussion